Packs/GoogleThreatIntelligence/Integrations/GoogleThreatIntelligence/GoogleThreatIntelligence.py

```diff
@@ -0,0 +1,2695 @@
+    elif resource_type == 'ip':
+        raise_if_ip_not_valid(resource)
+        raw_response = client.ip(resource)
+        if raw_response.get('error', {}).get('code') == 'QuotaExceededError':
+            return build_quota_exceeded_ip_output(client, resource)
+        if raw_response.get('error', {}).get('code') == 'NotFoundError':
+            return build_unknown_ip_output(client, resource)
+        indicator = _get_ip_indicator(client, score_calculator, resource, raw_response)
+    elif resource_type == 'url':
+        raw_response = client.url(resource)
+        if raw_response.get('error', {}).get('code') == 'QuotaExceededError':
+            return build_quota_exceeded_url_output(client, resource)
+        if raw_response.get('error', {}).get('code') == 'NotFoundError':
+            return build_unknown_url_output(client, resource)
+        indicator = _get_url_indicator(client, score_calculator, resource, raw_response)
+    elif resource_type == 'domain':
+        raw_response = client.domain(resource)
+        if raw_response.get('error', {}).get('code') == 'QuotaExceededError':
+            return build_quota_exceeded_domain_output(client, resource)
+        if raw_response.get('error', {}).get('code') == 'NotFoundError':
+            return build_unknown_domain_output(client, resource)
+        indicator = _get_domain_indicator(client, score_calculator, resource, raw_response)
+    else:
+        raise DemistoException(f'Could not find resource type of "{resource_type}"')
+
+    data = raw_response.get('data', {})
+    data.pop('relationships', None)
+    gti_assessment = data.get('attributes', {}).get('gti_assessment', {})
+
+    if data:
+        if gti_assessment:
+            data['attributes'] = {'gti_assessment': gti_assessment}
+        else:
+            data.pop('attributes', None)
+
+    return CommandResults(
+        f'{INTEGRATION_ENTRY_CONTEXT}.Assessment',
+        'id',
+        indicator=indicator,
+        readable_output=tableToMarkdown(
+            f'Google Threat Intelligence assessment of {resource_type}: "{resource}"',
+            {
+                'threat_score': gti_assessment.get('threat_score', {}).get('value'),
+                'severity': gti_assessment.get('severity', {}).get('value'),
+                'verdict': gti_assessment.get('verdict', {}).get('value'),
+            },
+            headers=[
+                'threat_score',
+                'severity',
+                'verdict',
+            ],
+            headerTransform=string_to_table_header,
+        ),
+        outputs=data,
+        raw_response=raw_response,
+    )
+
+
+def main(params: dict, args: dict, command: str):
+    results: Union[CommandResults, str, List[CommandResults]]
+    handle_proxy()
+    client = Client(params)
+    score_calculator = ScoreCalculator(params)
+
+    ip_relationships = (','.join(argToList(params.get('ip_relationships')))).replace('* ', '').replace(' ', '_')
```

Comment:
For cleanliness, consider moving the data parsing to a helper function and then just pass to it the param.

Reply:
Done!